akka-cluster-tools/src/main/resources/reference.conf

```diff
@@ -197,8 +197,9 @@ akka.cluster.singleton-proxy {
   # The actor name of the singleton actor that is started by the ClusterSingletonManager
   singleton-name = ${akka.cluster.singleton.singleton-name}
   
-  # The role of the cluster nodes where the singleton can be deployed. 
-  # If the role is not specified then any node will do.
+  # The role of the cluster nodes where the singleton can be deployed.
+  # Corresponding to the role used by the `ClusterSingletonManager`. If the role is not
+  # specified it's a singleton among all nodes in the cluster.
```

Comment:
Note that it must be the same for the singleton and the proxy as well?

Reply:
ok, added some more clarification

Reply:
👍